cmd/operator/operator.go

```diff
@@ -247,11 +257,57 @@ func run(c *cli.Context) {
 	}
 }
 
+func getObjects(objectKinds string) []client.Object {
+	var objs []client.Object
+
+	arr := strings.Split(objectKinds, ",")
+	for _, str := range arr {
+		switch str {
+		case "ConfigMap":
+			objs = append(objs, &v1.ConfigMap{})
+		case "Secret":
+			objs = append(objs, &v1.Secret{})
+		case "Pod":
+			objs = append(objs, &v1.Pod{})
+		case "Event":
+			objs = append(objs, &v1.Event{})
+		case "Service":
+			objs = append(objs, &v1.Service{})
+		case "ServiceAccount":
+			objs = append(objs, &v1.ServiceAccount{})
+		case "PersistentVolume":
+			objs = append(objs, &v1.PersistentVolume{})
+		case "PersistentVolumeClaim":
+			objs = append(objs, &v1.PersistentVolumeClaim{})
+		case "Deployment":
+			objs = append(objs, &appsv1.Deployment{})
+		case "DaemonSet":
+			objs = append(objs, &appsv1.DaemonSet{})
+		case "StatefulSet":
+			objs = append(objs, &appsv1.StatefulSet{})
+		case "ClusterRole":
+			objs = append(objs, &rbacv1.ClusterRole{})
+		case "ClusterRoleBinding":
+			objs = append(objs, &rbacv1.ClusterRoleBinding{})
+		case "Role":
+			objs = append(objs, &rbacv1.Role{})
+		case "RoleBinding":
+			objs = append(objs, &rbacv1.RoleBinding{})
+		case "StorageClass":
+			objs = append(objs, &storagev1.StorageClass{})
+		}
```

Comment:
nit: better to add else, if the type is not supported we should print a log

Reply:
sounds good